src/components/ProviderLogin.svelte

```diff
@@ -0,0 +1,139 @@
+<script lang="ts">
+  import { onMount } from 'svelte';
+
+  import type { WsProvider, ApiPromise } from '@polkadot/api';
+  import type { web3Enable, web3Accounts } from '@polkadot/extension-dapp';
+  import { dotApi, isLoggedIn, transactionSigningAddress } from '$lib/stores';
+
+  import { defaultDotApi } from '$lib/storeTypes';
+
+  import DropDownMenu from '$components/DropDownMenu.svelte';
+  import BlockSection from '$components/BlockSection.svelte';
+  import Button from '$components/Button.svelte';
+  import { pageContent } from '$lib/stores/pageContentStore';
+  import { Network, allNetworks, networkToInfo, selectedNetwork } from '$lib/stores/networksStore';
+  import { getApi, updateConnectionStatus } from '$lib/polkadotApi';
+  import { fetchAccounts, storeProviderAccounts } from '$lib/stores/accountsStore';
+
+  let wsProvider: WsProvider;
+  let thisWeb3Enable: typeof web3Enable;
+  let thisWeb3Accounts: typeof web3Accounts;
+
+  let networks = allNetworks();
+
+  let selectedNetworkAsString: string = '';
+  let customNetwork: string = '';
+
+  let thisDotApi = defaultDotApi;
+  dotApi.subscribe((api) => (thisDotApi = api));
```

Comment:
refer to the simpler way of subscribing described above.

Reply:
This is inherited logic.  We can change it later once everything is working.